dream_layer_backend/img2img_workflow.py

```
@@ -0,0 +1,258 @@
+    }
+    
+    # Determine model type based on model_name
+    model_name = data.get('model_name', 'v1-6-pruned-emaonly-fp16.safetensors')
+    if 'bfl' in model_name.lower() or 'flux' in model_name.lower():
+        workflow_request['model_name'] = 'bfl'
+    elif 'ideogram' in model_name.lower():  # Added check for ideogram models
+        workflow_request['model_name'] = 'ideogram'
+    else:
+        workflow_request['model_name'] = 'local'
+    
+    print(f"\nWorkflow request dictionary:")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)